### PR TITLE
Fix/issue#124

### PR DESCRIPTION
**Pull request by erisara**

### CODE REVIEW

File: src/main/webapp/app/user-dates/user-dates.component.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    $scope.onItemClick = function( item ) {
+        item.active = !item.active;
+        if (item.active) {
+            item.activeContent = item.content;
+        } else {
+            item.activeContent = item.shortContent;
+        }
+    }
+
+    function setTestData() {
```
Comment (dionisisvas) [inline src/main/webapp/app/user-dates/user-dates.component.js]: Is this needed? (the testItemList)

**Review by dionisisvas — CHANGES_REQUESTED** [review]

**Review by chris-iri — CHANGES_REQUESTED** [review]
Duplicate - ignore

**Review by chris-iri — APPROVED** [review]

**Review by chris-iri — CHANGES_REQUESTED** [review]
The `user-dates` component should be added inside `profile.template.html` located at */app/views/
The `user-profile` component is unrelated and the same code shouldn't be copied inside it.

File: src/main/webapp/app/user-profile/user-profile.template.html [hunk 2]
```diff
@@ -67,8 +67,6 @@
         </p>
       </md-card-content>
     </md-card>
-  </div>
```
Comment (dionisisvas) [inline src/main/webapp/app/user-profile/user-profile.template.html:70]: Is this </div> tag need to be deleted or was it deleted by mistake?

**Review by dionisisvas — APPROVED** [review]

**Review by dionisisvas — APPROVED** [review]

**Comment by dionisisvas** [comment]
Closes Issue #124

**Pull request merged**